### PR TITLE
general layout prototype

### PR DESCRIPTION
**Pull request by briceaceugeniu**

### CODE REVIEW

File: web/src/App.tsx [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
+import TaskCard from "./components/TaskCard";
+import SupertasksList from "./components/supertasks/SupertasksList";
 
 function App() {
-  let longtext = "";
+  const [tasksCount, setTasksCount] = useState(1);
 
-  for (let i = 0; i < 3; i++) {
-    longtext += "some lorem ipsum thingie going on.";
+  let longtext = [];
```
Comment (bmustiata) [inline web/src/App.tsx]: `longtext`? `longtext.push(<TaskCard..` ??

File: web/src/App.tsx [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
-                onAddClick={() => { alert('add') }}
-                onEditClick={() => { alert('edit') }}
-                percentage={10}
-                />
-    </Container>
+    <div className="grid-container">
+      <div className="nav demo">
+        <div>Navbar</div>
+      </div>
+      <div className="task-tree demo">{longtext}</div>
```
Comment (bmustiata) [inline web/src/App.tsx]: This should be its own component, so we can change the layouting later.

**Review by bmustiata — CHANGES_REQUESTED** [review]

File: web/src/App.css [hunk 3]
```diff
@@ -1,38 +1,41 @@
...
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.grid-container {
+  display: grid;
+  grid-template-areas:
+        'nav nav nav nav nav nav'
+        'task-tree task-tree task-tree task-tree task-tree task-list'
```
Comment (bmustiata) [inline web/src/App.css:9]: what is this?

**Review by bmustiata — CHANGES_REQUESTED** [review]

File: web/src/App.css (hunk 3 above)
Comment (briceaceugeniu) [reply web/src/App.css:9]: https://developer.mozilla.org/de/docs/Web/CSS/grid-template-areas

**Review by bmustiata — APPROVED** [review]

**Pull request merged**